include/seqan/modifier/modifier_view.h

```diff
@@ -475,6 +475,24 @@ convert(TSequence const & sequence, TFunctor const &F)
 #endif
 }
 
+// --------------------------------------------------------------------------
+// Function assign()
+// --------------------------------------------------------------------------
+
+template <typename THost, typename TFunctor>
+void assign(ModifiedString<THost const, ModView<TFunctor> > & target,
+            THost const & host)
+{
+    target._host = &host;
+}
```

Comment:
This is the same signature as the non-const version below if THost is const. Probably this results in an ambiguous function call error? What was the intension of the const version anyway?